lib/improver/cli/nowcast_extrapolate.py

```diff
@@ -173,17 +76,18 @@ def process(input_cube, u_cube=None, v_cube=None, speed_cube=None,
         ValueError:
             can either use speed_cube and direction_cube or u_cube and v_cube.
```

Comment:
Suggest:
```
Returns:
    iris.cube.Cube:
        Cube containing extrapolation forecasts at all required lead times
```
The `ValueError` is no longer raised as you have removed this option.

Reply:
Already done on my local version